src/osw_config_keys.cpp

```diff
@@ -50,6 +52,9 @@ OswConfigKeyPassword fallbackWifiPass1st("b1", "WiFi", "2nd Password", nullptr,
 OswConfigKeyString fallbackWifiSsid2nd("a2", "WiFi", "3rd SSID", "Leave empty to disable", CONFIG_FALLBACK_2ND_WIFI_SSID);
 OswConfigKeyPassword fallbackWifiPass2nd("b2", "WiFi", "3rd Password", nullptr, CONFIG_FALLBACK_2ND_WIFI_PASS);
 #endif
+#ifdef OSW_FEATURE_BLE_SERVER
+OswConfigKeyBool bleBootEnabled("f", "System", "Enable BLE on boot", "This will drain your battery faster!", BLE_ON_BOOT);
```

Comment:
AFAIK, we support web GUI i8n for decription section...right?

Reply:
Hmmm... Not sure, the bytes are transferred, but the encoding is not specified, hence the browser should just use ASCII.